api/resolvers/wallet.js

```diff
@@ -551,14 +574,25 @@ const resolvers = {
     },
     preimage: async (withdrawl, args, { lnd }) => {
       try {
-        if (withdrawl.status === 'CONFIRMED') {
+        if (withdrawl.status === 'CONFIRMED' && withdrawl.hash) {
           return withdrawl.preimage ?? (await getPayment({ id: withdrawl.hash, lnd })).payment.secret
         }
       } catch (err) {
         console.error('error fetching payment from LND', err)
       }
     }
   },
+  Direct: {
+    nostr: async (direct, args, { models }) => {
```

Comment:
We don't send zap notifications for direct payments, but we do display them if you navigate to the payment from satistics.

We store them so that if we eventually add preimage gathering we can broadcast the zaps.